src/Security/Security.php

```diff
@@ -1163,7 +1163,7 @@ public static function default_admin_username()
     /**
      * Get default admin password
      *
-     * @deprecated 4.0.0:5.0.0 Use DefaultAdminService::getDefaultAdminPassword()
+     * @deprecated 4.0.0 Use DefaultAdminService::getDefaultAdminPassword()
```

Comment:
Update the notice message and version as well.
```suggestion
     * @deprecated 4.0.0 Use DefaultAdminService::getDefaultAdminPassword() instead
```

Reply:
Updated